tests/test-cases/Group1-Docker-Commands/1-26-Docker-Hello-World.robot

```diff
@@ -1,14 +1,11 @@
 *** Settings ***
 Documentation  Test 1-26 - Docker Hello World
 Resource  ../../resources/Util.robot
-#Suite Setup  Install VIC Appliance To Test Server
-#Suite Teardown  Cleanup VIC Appliance On Test Server
+Suite Setup  Install VIC Appliance To Test Server
+Suite Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Hello world
-    ${status}=  Get State Of Github Issue  1578
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-26-Docker-Hello-World.robot needs to be updated now that Issue #1578 has been resolved
-    Log  Issue \#1578 is blocking implementation  WARN
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} run hello-world
-    #Log  ${output}
-    #Should Be Equal As Integers  ${rc}  0
\ No newline at end of file
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} run hello-world
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
we can't use -it here and -i is not guaranteed to give an output :/ I can check logs output